nodejs/nodejs-cloud-run-hello-world/index.js

```diff
@@ -0,0 +1,11 @@
+const app = require('./app');
+const pkg = require('./package');
+const PORT = process.env.PORT || 8080;
+
+app.listen(PORT, () => {
+  console.log(
+    'Hello from Cloud Run! The container started successfully and is listening for HTTP requests on $PORT'
```

Comment:
I've tended to have the `.listen` in the same file as the `app`, or inverted things a bit, where `app` would expose methods, and the harness would just wire up the routes.

I can see the advantage to this approach, if you intend to add a couple additional routes on to app; curious why you like this design approach?

Reply:
It's a legacy of using supertest, where injecting requests into the app object is part of the API. I'd probably do as you say and and have app.js just expose methods, but from a sample standpoint I think it confuses reading individual slices of code to aggressively separating routing from logic.

Reply:
I'll go ahead and combine them into one file.